CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+* [Enhancement] Support latest version of boto3; include the 
+  `request_checksum_calculation` config option.
```

Comment:
This changelog doesn't reflect the correct name of the config option; it should say `S3_REQUEST_CHECKSUM_CALCULATION` instead.